receiver/jmxreceiver/receiver.go

```diff
@@ -87,7 +84,7 @@ func (jmx *jmxMetricReceiver) Start(ctx context.Context, host component.Host) er
 	jmx.configFile = tmpFile.Name()
 	subprocessConfig := subprocess.Config{
 		ExecutablePath: "java",
-		Args:           append(jmx.config.parseProperties(jmx.logger), jmxMainClass, "-config", jmx.configFile),
+		Args:           append(jmx.config.parseProperties(jmx.logger), jmx.config.jarMainClass(), "-config", jmx.configFile),
```

Comment:
@SylvainJuge  The receiver is always launching the JAR as classpath and specifing the JAR's main class, would that cover the additional JARs launch detailed here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/40954/files#r2171140179?